packages/expo-notifications/src/Notifications.types.ts

```diff
@@ -268,77 +314,42 @@ export type CalendarTriggerInputValue = {
 };
 
 /**
- * A trigger that will cause the notification to be delivered once or many times when the date components match the specified values.
- * Corresponds to native [`UNCalendarNotificationTrigger`](https://developer.apple.com/documentation/usernotifications/uncalendarnotificationtrigger?language=objc).
- * @platform ios
+ * See [`SchedulableNotificationTriggerInput`](#schedulablenotificationtriggerinput)
  */
-export type CalendarTriggerInput = CalendarTriggerInputValue & {
+export type SchedulableNotificationTriggerInputObject = {
```

Comment:
would the generated docs look bad if this were a discriminating union of `Native*TriggerInput`s?

It could make the code, and the public api a bit better but not sure about docs.

Reply:
Let's discuss what you have in mind here...

Reply:
Ok I've reworked this as a union, and renamed `CalendarInputTypes` to `SchedulableTriggerInputTypes`.